cmd/minikube/cmd/start_flags.go

```diff
@@ -98,6 +99,7 @@ const (
 	nodes                   = "nodes"
 	preload                 = "preload"
 	deleteOnFailure         = "delete-on-failure"
+	dockerBaseImage         = "docker-base-image"
```

Comment:
change this to "base-image" since this is for both docker or podman.

Reply:
done